apps/sqlparser/sqlparser.py

```diff
@@ -0,0 +1,178 @@
+import argparse
+import shutil
+import csv
+import sys
+
+from datetime import datetime
+from pathlib import Path
+from collections import Counter
+
+# constants, can be overriden using arguments
+
+## implied folder structure: modules/*/sql/dbname/*/*.sql
+glob = "**/sql/**/*.sql"
+
+repo_folder = Path("/Users/v0/Py/_hobby/wow/server")
+
+
+default_argparsed_args = ["--help"]
+# default_argparsed_args = "-y", ""  # , "--glob", "test"
+
+source_suffix = "modules"
+source_dir = lambda: repo_folder / source_suffix
+target_suffix = "data/sql/custom"
+target_dir = lambda: repo_folder / target_suffix
+report_suffix = "modules_sql_parsed.csv"
+report_path = lambda: repo_folder / report_suffix
+
+modules_to_ignore = ["mod-playerbots"]
+
+counters = dict(
+    written=0, skipped=0, total=0, modules=Counter()  # module: counter
+)
+
+
+def get_db_name(arg):
+    if arg.startswith("db"):
+        # assumes db_*.sql or db-*.sql filename
+        arg = arg[3:]
+    return f"db_{arg}"
+
+
+def collect_sql_files(full_fp, files_to_ignore=None, dry=False):
```

Comment:
default arg for files_to_ignore can be an empty list so it's more clear as to what should be passed in

beware, common footgun with a list as a default argument is that all function invocations share the same list (as in, the pointer to that list), but as long as you're not doing anything to the list (adding/removing things) that's fine. 

```suggestion
def collect_sql_files(full_fp, files_to_ignore=[], dry=False):
```

Reply:
default arg as list is not allowed by some PEP and flake, i dont see big reason to violate it here.
are you sure we need this change?